tests/integration/events.bats

```diff
@@ -111,3 +111,56 @@ function teardown() {
   run eval "grep -q 'test_busybox' events.log"
   [ "$status" -eq 0 ]
 }
+
+@test "events oom " {
+  # XXX: currently cgroups require root containers.
+  requires root
+  init_cgroup_paths
+
+  # Set some initial known values
+  if [ "$CGROUP_UNIFIED" == "yes" ]; then
```

Comment:
Let's merge https://github.com/opencontainers/runc/pull/2370 first so we won't have to have this switch here.